pkg/indicator/ema.go

```diff
@@ -0,0 +1,60 @@
+package indicator
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/store"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type EWMA struct {
+	Interval types.Interval
+	Window   int
+	Values   Float64Slice
+	EndTime  time.Time
+}
+
+func (inc *EWMA) calculateAndUpdate(kLines []types.KLine) {
+	if len(kLines) < inc.Window {
+		// we can't calculate
+		return
+	}
+
+	var index = len(kLines) - 1
+	var lastK = kLines[index]
+	var multiplier = 2.0 / float64(inc.Window+1)
```

Comment:
Could you add ref link to this calculation settings?

Reply:
sure